### PR TITLE
Set page titles per page

### PR DESCRIPTION
**Pull request by Sperling-0**
- Set page titles per page
- Add temporary file names to git ignore

Ticket: https://phabricator.wikimedia.org/T361797

### CODE REVIEW

File: i18n/de_DE/messages/siteMetadata.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Spenden für Freies Wissen - Wikimedia Deutschland",
+  "title": "Spenden für Freies Wissen - Wikimedia Deutschland Fundraising",
```
Comment (moiikana) [inline i18n/de_DE/messages/siteMetadata.json]: just saw that, didn't look where it's used yet but we probably should have a German word here in the de_DE keys

**Comment by gbirke** [comment]
I've added @KaiNissen to provide page names

File: i18n/de_DE/messages/siteMetadata.json (hunk 1 above)
Comment (Sperling-0) [reply i18n/de_DE/messages/siteMetadata.json]: Yes yes, the word I added is just a place holder.

**Comment by Abban** [comment]
We should also be replacing the `Donation to free knowledge`/`Spenden für Freies Wissen` parts with the actual page names, so that should be changed to a replaceable template tag.

Also keep in mind that we should be keeping the total length of title tag short, it's easier to understand, fits better into browser tabs, plus more than 60-70 characters will make it truncated in search results

File: i18n/en_GB/messages/messages.json [hunk 2]
```diff
@@ -464,6 +464,27 @@
...
+  "donation_form_page_title": "Donation form",
+  "error_page_title": "Error",
+  "faq_page_title": "Frequently Asked Questions",
+  "membership_application_confirmation_page_title": "Membership application confirmed",
+  "membership_application_page_title": "Membership application form",
+  "page_not_found_page_title": "Page not found",
+  "privacy_protection_page_title": "Privacy protection",
+  "subscription_confirmation_page_title": "E-Mail address confirmed",
+  "supporters_page_title": "Hall of Fame",
+  "static_page_title": "Static Page",
```
Comment (KaiNissen) [inline i18n/en_GB/messages/messages.json]: Is this shown for every static page? Or is it replaced by an actual page title at a later step?

Comment (Sperling-0) [reply i18n/en_GB/messages/messages.json]: Sorry, this is a leftover code. I will remove it.
Now, we set the title for static pages dynamically like this:
```javascript
pageTitle: staticPage.getAttribute( 'data-page-title' ),
```

**Pull request merged**